skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -108,8 +109,8 @@ cdef inline np_floats patch_distance_3d(np_floats [:, :, :] p1,
     return _fast_exp(-max(0.0, distance))
 
 
-def _nl_means_denoising_2d(cnp.ndarray[np_floats, ndim=3] image, Py_ssize_t s=7,
-                           Py_ssize_t d=13, double h=0.1, double var=0.):
+def _nl_means_denoising_2d(cnp.ndarray[np_floats, ndim=3] image, Py_ssize_t s,
+                           Py_ssize_t d, double h, double var):
```

Comment:
it's ok to remove default values here since the defaults are given in the Python function and passed to the Cython one.